assets/core/css/custom.css

```
@@ -1,6 +1,578 @@
 /* =============================================================================
-   InvoicePlane
-   Custom Stylesheet
-
+   InvoicePlane Custom Stylesheet
    All changes made in this file will override the default styles.
-   ========================================================================== */
\ No newline at end of file
+   ============================================================================*/
```

Stylelint (17.7.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)